### ISSUE TITLE
Connecting to a host only providing /api

### ISSUE THREAD

**Issue by lomenzel** [JHubi1/ollama-app#38]
The settings page checks if the response to https://example.com is "Ollama is running". i have a setup where example.com is hosting an ollama web frontend and the reverseproxy proxypasses example.com/api to the ollama service. the result is, that the ollama app refuses the host, but it could use it

**Comment by Pietro395** [comment]
I have the same problem

**Comment by kudos** [comment]
I hacked this together for myself. If you temporarily reverse proxy both the root path `/` and the ollama path `/ollama/` to your Ollama instance you can convince the app to accept the settings. Then restore the reverse proxying to openwebui and it will continue working.

**Comment by Badlee2020** [comment]
I agree this would make this app a lot more practical. I use openwebui to consolidate all my various ai accounts and local LLMs. It would be amazing to have that all in one phone app
